assets/js/implicit-login.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    if ( ! window.location.hash || window.location.hash.indexOf('id_token') < 0 ) {
+        return;
+    }
+
+    var hash = window.location.hash;
+    if (hash[0] === '#') {
```

Comment:
will this work fine if `hash=""`?

Reply:
![screenshot 2018-04-04 11 57 59](https://user-images.githubusercontent.com/855223/38328326-751a6c74-37ff-11e8-9145-7ef8611a4e69.png)

Reply:
But it can't be empty because we've escaped already if that's the case